gemviz/mainwindow.py

```diff
@@ -147,25 +143,24 @@ def clearContent(self, clear_cat=True):
     def serverList(self):
         return self._serverList
 
-    def setServerList(self, uri_list=None):
-        """Set the list of server URIs and remove duplicate"""
-        unique_uris = set()
-        new_server_list = []
-
-        if not uri_list:
-            previous_uri = settings.getKey(TILED_SERVER_SETTINGS_KEY)
-            candidate_uris = ["", previous_uri, TESTING_URL, LOCALHOST_URL, "Other..."]
+    def setServerList(self, selected_uri=None):
```

Comment:
Now, this code does not take a list as argument and it does not set the list.  It adds the URI to the list.  The method name could be more representative of what it does: `addServerUri()`.  Update the docstring, too.